lib/tags.js

```diff
@@ -87,19 +88,25 @@ exports.parseTagLine = (line) => {
  * console.log(footer.Bar);
  * > baz
  */
-exports.parseFooterTagLines = (footerTagLines) => {
+exports.parseFooterTagLines = (footerTagLines, lowerCaseFooterTags) => {
   return _.chain(footerTagLines)
     .reject((line) => {
       return _.isEmpty(line.trim());
     })
     .map((tagLine) => {
       const tag = exports.parseTagLine(tagLine);
+      const finalTags = [];
 
-      return [
-        tag.key,
-        tag.value
-      ];
+      finalTags.push([ tag.key, tag.value ]);
```

Comment:
I haven't actually test this, but I believe this whole thing can be simplifies as follows:

```js
.map((tagLine) => {
  const tag = exports.parseTagLine(tagLine);

  if (lowerCaseFooterTags) {
    tag.key = tag.key.toLowerCase();
  }

  return [
    tag.key,
    tag.value
  ];
})
```

This also has the advantage that `.flatten()` is not needed anymore.

If you prefer a more concise functionally-pure style:

```js
.map((tagLine) => {
  const tag = exports.parseTagLine(tagLine);

  return [
    lowerCaseFooterTags ? tag.key.toLowerCase() : tag.key,
    tag.value
  ];
})
```